src/AccessibilityInsights.SharedUx/Dialogs/TelemetryApproveContainedDialog.xaml.cs

```diff
@@ -55,9 +55,9 @@ private void btnExit_Click(object sender, RoutedEventArgs e)
             DialogResult = ckbxAgreeToHelp.IsChecked ?? false;
 
             if (DialogResult)
-                TelemetryController.EnableTelemetry();
```

Comment:
This is just a rename to better distinguish which level of enabling we are setting